sklearn/covariance/shrunk_covariance_.py

```diff
@@ -355,6 +354,9 @@ class LedoitWolf(EmpiricalCovariance):
 
     Attributes
     ----------
+    location_ : array-like, shape (n_features,)
+        Estimated robust location
```

Comment:
Thanks for the changes @andreanr. As I understand the code, `LedoitWolf` doesn't try to do any _robust_ estimation of the mean/location. For the ones which compute a simple mean as an estimate for `location_`, we've used this description: "Estimated location, i.e. the estimated mean." ([example](http://scikit-learn.org/dev/modules/generated/sklearn.covariance.ShrunkCovariance.html#sklearn.covariance.ShrunkCovariance))

Reply:
That's true, I'll change it. Thanks.